app/models/setting/content.rb

```diff
@@ -60,7 +60,9 @@ def self.load_defaults
                                               "such as virtual machines and DNS records may also be deleted."),
                  false, N_('Delete Host upon unregister')),
         self.set('register_hostname_fact', N_("When registering a host via subscription-manager, force use the specified fact (in the form of 'fact.fact')"),
-                 '', N_('Subscription manager name registration fact'), nil)
+                 '', N_('Subscription manager name registration fact'), nil),
+        self.set('erratum_install_batch_size', N_("Force pulp to install errata in batches. Set to 0 to install all errata in one batch."),
```

Comment:
I might reword this to "Errata installed via katello-agent will be triggered in batches of this size".  No real need to mention pulp, but its important to mention that its katello-agent (versus remote-execution)